matter/wifi/wf200/efr_spi.c

```diff
@@ -74,6 +74,10 @@ static bool spi_enabled = false;
 
 uint8_t wirq_irq_nb = SL_WFX_HOST_PINOUT_SPI_IRQ; // SL_WFX_HOST_PINOUT_SPI_WIRQ_PIN;
 
+#define LENGTH_0        	0
```

Comment:
This define isn't really usefull.

Reply:
This isn't resolved, define should be removed.

Reply:
done